MatrixSDK/Data/MXRoomSummary.h

```diff
@@ -91,6 +91,11 @@ FOUNDATION_EXPORT NSString *const kMXRoomSummaryDidChangeNotification;
  */
 @property (nonatomic, readonly) MXRoom *room;
 
+/**
+ Shortcut to the corresponding room type.
+ */
+@property (nonatomic) NSString *roomType;
```

Comment:
It may be wiser to return a `MXRoomType` here instead of plain string. So we can handle default value handlings inside `MXRoomSummary`, so that way Swift users would get an enum.

Reply:
In fact we can support any room type values. When there is no value for the room type it can be interpreted differently on each client. So it's difficult to not constraint the property plus support both ObjC and Swift atm